src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -56,6 +56,18 @@ public RangeConstraint(IComparable from, IComparable to) : base( from, to )
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="RangeConstraint"/> class for objects not implementing IComparable. A class implementing IComparable must be passed through Using for a successful test.
+        /// </summary>
+        /// <remarks>A class implementing IComparable must be passed through Using for a successful test. from must be less than or equal to true</remarks> 
+        /// <param name="from">Inclusive beginning of the range. Must be less than or equal to to.</param>
+        /// <param name="to">Inclusive end of the range. Must be greater than or equal to from.</param>
+        public RangeConstraint(object from, object to) : base(from, to)
+        {
+            this.from = from;
+            this.to = to;
+        }
+
```

Comment:
It seems that this constructor doesn't actually create a usable RangeConstraint, since a Using must be provided. Two options seem possible: 1) Add the comparer to the constructor or 2) construct some other type, which implements Using and returns a range constraint when Using is provided.